sdk/core/azure-core-amqp/src/main/java/com/azure/core/amqp/implementation/ExtendedAmqpMessage.java

```diff
@@ -0,0 +1,40 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+package com.azure.core.amqp.implementation;
+
+import org.apache.qpid.proton.message.Message;
+import org.apache.qpid.proton.message.impl.MessageImpl;
+
+import java.nio.ByteBuffer;
+
+/***
+ *  Keep additional information for example delivery tag which represent lock token.
+ */
+
+public class ExtendedAmqpMessage extends MessageImpl {
```

Comment:
Could this potentially use a better name? `ExtendedAmqpMessage` is very vague.

Reply:
removed it